components/post_view/post_edited_indicator/post_edited_indicator.tsx

```diff
@@ -52,34 +53,71 @@ const PostEditedIndicator = ({postId, isMilitaryTime, timeZone, editedAt = 0}: P
         relativeOrDate: date,
         time,
     });
+    const viewHistoryText = formatMessage({
```

Comment:
Same here.  You can move this outside the component function.

Reply:
I can not move this outside of the component because `formatMessage` is inside the function and coming from `useInt`